cmd/fleet/main.go

```diff
@@ -92,7 +92,7 @@ func initLogger(cfg *config.Config, version, commit string) (*logger.Logger, err
 		Int("ppid", os.Getppid()).
 		Str("exe", os.Args[0]).
 		Strs("args", os.Args[1:]).
-		Msg("boot")
+		Msg("Fleet server logger initialised")
```

Comment:
@scunningham "Boot fleet server"? There is one problem in Cloud that it is not directly visible if it is an elastic-agent or fleet-server log. Maybe we should add a component or something to each log message?

Reply:
You can see it is from the fleet server in one of the other fields. but yeah it is tricky.
We could formalize a component type field, we use it haphazardly in a couple places
context: fleetserver.enroll
Or some such.